barretenberg/cpp/src/barretenberg/crypto/pedersen_hash/pedersen_refactor.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+// TODO(@zac-wiliamson #2341 rename to pedersen.hpp once we migrate to new hash standard)
+
+#include "../generators/fixed_base_scalar_mul.hpp"
+#include "../generators/generator_data.hpp"
+#include "../pedersen_commitment/pedersen_refactor.hpp"
+#include "barretenberg/ecc/curves/grumpkin/grumpkin.hpp"
+#include <array>
+
+namespace crypto {
+
+/**
+ * @brief Performs pedersen hashes!
+ *
+ * To hash to a size-n list of field elements `x`, we return the X-coordinate of:
+ *
+ *      Hash(x) = n.[h] + Commit(x)
+ *
+ * Where `g` is a list of generator points defined by `generator_data`
```

Comment:
nit: Theres no g in the above formula -- I'm assuming in a previous version it said n.[h] + x_0. [g_0] + x_1 . [g_1] +... + x_n . [g_n]

Reply:
fixed